src/dl.cpp

```diff
@@ -68,7 +70,25 @@ dl_error_t dl_instance_load( dl_ctx_t             dl_ctx,          dl_typeid_t
 	DL_ASSERT( (uint8_t*)instance + header->instance_size > packed_instance || (uint8_t*)instance < packed_instance + header->instance_size );
 	memcpy( instance, packed_instance + sizeof(dl_data_header), header->instance_size );
 
-	dl_internal_patch_instance( dl_ctx, root_type, (uint8_t*)instance, 0x0, (uintptr_t)instance );
+	if( header->version == 1 )
```

Comment:
I wonder if it is worth keeping the backwards compat... I don't want DL to turn out to be as riddled with backwards-compat code as "some other lib we know". I know it served a purpose there but do you need it? @lundmark might be able to answer if some other users than you would need it.

Reply:
Answered above